docs/BreakingChanges/146 CurrentCulture flows across Tasks.md

```diff
@@ -24,10 +24,8 @@ AppContext.SetSwitch("Switch.System.Globalization.NoAsyncCurrentCulture", true);
 ```
 
 ### Affected APIs
-* `M:System.Globalization.CultureInfo.set_CurrentCulture(System.Globalization.CultureInfo)`
```

Comment:
Again, I'm not sure that we need to make this change because I think users who only get the CurrentCulture are less likely to care about this.